en-US/Players_Guide/03_Creating_a_Character/Languages.md

```diff
@@ -29,5 +29,4 @@ _Languages (table)_
 ___
 
 **Signing.**
-You must have at least one hand free to communicate by sign, and the creature you are communicating with must be able to see you.
-When attempting to make subtle signs, to remain unnoticed you must succeed on a Sleight of Hand check against the passive Perception scores of observers.
+Sign languages are independent languages with their own grammars and vocabularies. If your character knows a sign language, they must learn it separately from a spoken or written language unless the campaign setting includes a signed form of that language. You must have at least one hand free to communicate by sign, and the creature you are communicating with must be able to see you. When attempting to make subtle signs to remain unnoticed, you must succeed on a Sleight of Hand check against the passive Perception of observers.
```

Comment:
Please add each sentence in a separate line. That's how it's done everywhere else in this repository.